php-frontend/src/test/java/org/sonar/php/tree/impl/expression/AnonymousClassTreeTest.java

```diff
@@ -45,15 +58,22 @@ public void test() throws Exception {
   }
 
   @Test
-  public void test_with_named_arguments() throws Exception {
+  void test_with_named_arguments() {
```

Comment:
Not related to this PR directly, I see you changed some other part of this test class (junit 4 -> 5), I would also adap the method name from snake_case to camelCase.